x/merkledb/codec.go

```diff
@@ -89,6 +81,7 @@ func encodedDBNodeSize(n *dbNode) int {
 // allocations.
 func hashNode(n *node) ids.ID {
 	var (
+		// sha.Write always returns nil, so we ignore its return values.
```

Comment:
This seems like it's in the wrong place

Reply:
I put the comment here (where we define `sha`) rather than commenting it on each of the calls to `sha.Write`